src/state/reducers/members.js

```diff
@@ -0,0 +1,42 @@
+import { createAction, createReducer } from 'redux-act';
+import { ActionType } from 'redux-promise-middleware'
+
+import { DBCollections, Errors } from '../../constants'
+
+const { db } = window
+
+const initialState = {
+  members: [],
+  loading: false,
+  editing: false,
+  error: null,
+}
+
+export const loadMembers = createAction('LOAD_MEMBERS', () => ({
```

Comment:
ok, this is a little beyond my knowledge of redux, but I see loadMembers is called by the members component on componentDidMount, doesn't it mean that firebase snapshot is called every time anyway?